src/index.js

```diff
@@ -0,0 +1,107 @@
+const state = {
+    temp: 72,
+    city: "Seattle",
+    lat: 47.608013,
+    long: -122.335167,
+};
+
+
+const convertKtoF = (temp) => {
+    return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const getWeather = () => {
+    let lat, long;
+    axios.get('http://127.0.0.1:5500/weather',
+        {
+            params: {
+                q: state.city,
```

Comment:
This was expecting you to pass `lat` and `lon` parameters. This causes you to get a response that says `{"message":"must provide lat and lon parameters"}` and makes your `.catch` run because `weather` ends up being undefined on line 23.

Reply:
```suggestion
                lat: state.lat,
                lon: state.long
```